adafruit_logging.py

```diff
@@ -2,6 +2,7 @@
 #
 # SPDX-License-Identifier: MIT
 
+# noinspection PyUnresolvedReferences
```

Comment:
`PyUnresolvedReferences` is a code inspection in Pycharm (by Jetbrains)

Reply:
Could you please remove this comment?